gravity.sh

```diff
@@ -239,17 +227,6 @@ database_table_from_file() {
     echo -e "  ${CROSS} Unable to remove ${tmpFile}"
 }
 
-# Update timestamp of last update of this list. We store this in the "old" database as all values in the new database will later be overwritten
-database_adlist_updated() {
```

Comment:
This function was removed but I don't see a corresponding function in https://github.com/pi-hole/FTL/pull/1559

Reply:
https://github.com/pi-hole/FTL/pull/1565